src/tools/compiletest/src/runtest.rs

```diff
@@ -1909,8 +1909,7 @@ impl<'test> TestCx<'test> {
         } else {
             Command::new(&self.config.rustdoc_path.clone().expect("no rustdoc built yet"))
         };
-        // FIXME Why is -L here?
-        rustc.arg(input_file); //.arg("-L").arg(&self.config.build_base);
+        rustc.arg(input_file);
```

Comment:
Why is this change part of this PR?

Reply:
Oh, I noticed it while looking for `{{src-base}}` and it looked super outdated. It's working fine without -L so I don't think the comment is necessary.